README.md

```diff
@@ -1,2 +1,29 @@
-# wg
-Coordination repository of the Game Development Working Group 
+# Rust game development working group
+
+The game development working group's main purpose is to make Rust the default language option for game developers.
+
+## What are we doing?
+
+At the very high level we have two main tasks:
+
+* To improve the experience of people using rust for game development. We want game developers to have a representation in the Rust project. We will be a bridge between them and the Rust team and make make visible the problems they face to the dev team and help in solving them.
+
+* Share knowledge and make game engine development more accessible for people that are not engine developers.
+
+## Goals for this year
+
+* Gather a list of all the main gaps that separate us from the ideal state described in the introduction. It could be tooling gaps, a language feature, a library feature (eg. custom allocators). After this is gathered, we can start working on the biggest issues in that list. There are a few initial candidates but we encourage you to tell us where you are having issues so we can help. It may involve helping out wiht a low-level library
```

Comment:
```suggestion
* Gather a list of all the main gaps that separate us from the ideal state described in the introduction. It could be tooling gaps, a language feature, a library feature (eg. custom allocators). After this is gathered, we can start working on the biggest issues in that list. There are a few initial candidates but we encourage you to tell us where you are having issues so we can help. It may involve helping out with a low-level library
```